drivers/pinmux/pinmux_stm32.c

```diff
@@ -375,9 +375,9 @@ int stm32_dt_pinctrl_remap(const struct soc_gpio_pinctrl *pinctrl,
 #if DT_NODE_HAS_STATUS(DT_NODELABEL(spi1), okay)
 	case DT_REG_ADDR(DT_NODELABEL(spi1)):
 		if (remap == REMAP_1) {
-			LL_GPIO_AF_EnableRemap_SPI1();
+			AFIO->MAPR |= AFIO_MAPR_SPI1_REMAP;
```

Comment:
Good catch.
Are there any other remap cases that should be fixed ?

Reply:
yes probably, all other Remap LL functions
Maybe this could be changed in the Stm32Cube LL( ../modules/hal/stm32/stm32cube/stm32f1xx/drivers/include/stm32f1xx_ll_gpio.h)
Second : the stm32_dt_pinctrl_remap() function starts with `LL_APB2_GRP1_EnableClock(LL_APB2_GRP1_PERIPH_AFIO);`
but does not ends with 
`LL_APB2_GRP1_DisableClock(LL_APB2_GRP1_PERIPH_AFIO);`
like in gpio_stm32_afio_init() function of the gpio_stm32.c
I think the clock should be disable after using the AFIO instance